org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -129,6 +131,8 @@
 
     private boolean firmwareUpdateInProgress = false;
 
+    private ExecutorService commandScheduler = Executors.newCachedThreadPool();
```

Comment:
With this change, a thread pool will be created for each zigbee thing handler instance. Shouldn't we rather use a shared pool for all zigbee thing handler instances, as follows (where the `ThreadPoolManager` is from package `org.eclipse.smarthome.core.common`), wdyt?

```
private ExecutorService commandScheduler = ThreadPoolManager.getPool("zigbee-thinghandler-commands");
```

As a side note, a colleague of mine mentioned that the thread pools returned by `newCachedThreadPool` turned out to be problematic in an OSGi setup, because they caused class loader leaks at least in our setting (but I don't know the details about those issues).